gsa/src/web/pages/cpes/detailspage.js

```diff
@@ -136,8 +138,10 @@ const Details = ({entity, links = true}) => {
               ))}
             </TableBody>
           </Table>
-        ) : (
+        ) : cve_refs === 0 ? (
```

Comment:
I guess cve_refs is a count and not an array right?

Reply:
Right, cves is the array with all the cves in in it and cve_refs is the count. I totally missed that at first. cve_refs is also used for showing the number of cves on the listpage.